src/lib/components/SearchBar/overriden/ResponsiveSearchBar.js

```diff
@@ -0,0 +1,176 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Input } from 'semantic-ui-react';
+import { screenIsWiderThan } from '@components/utils';
+import { Breakpoints } from '@components/Media';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.updateQueryString = this.props.updateQueryString;
+    this.state = {
+      currentValue: this.props.queryString || '',
+    };
+  }
+
+  onInputChange = (queryString) => {
```

Comment:
are these methods needed? aren't they implemented in RSK?

Reply:
Hmm, as I understand it, we override the entire RSK [searchBar](https://github.com/inveniosoftware/react-searchkit/blob/master/src/lib/components/SearchBar/SearchBar.js#L155), so we need all the methods? or not?

Reply:
Following @ntarocco suggestion I parameterized the RSK SearchBar and it works correclty. Thanks!